### DISCUSSION TITLE
No of cells in any row or column more than or equal to 7

### DISCUSSION THREAD

**Discussion by ap264** [neperfepx/neper#557] [Q&A] [answered]
Hi 

I would like to know if is possible to create a cube with voronoi cells of more than or equal to 7 in every row and column?

Thank you

**Comment by rquey** [comment, marked as answer]
Not sure I understand properly. Possible answers are
1. no, this is not among the possible inputs
2. yes, just choose a large enough number of cells:
```
neper -T -n 1000
neper -V n1000-id1.tess -print img
```
![image](https://user-images.githubusercontent.com/2659376/189827174-c89504d7-e327-439e-962a-399e00b168ef.png)
